o8g/Sets/Core Set/set.xml

```diff
@@ -2327,69 +2472,79 @@
       <property name="Type" value="Role"/>
       <property name="Traits" value="Keeper. Air."/>
       <property name="Text" value="Increase your deckbuilding influence value by 3.&#10;Reaction: After you win an [Air] conflict as the defending player - gain 1 fate."/>
+      <property name="Ring" value="Air"/>
```

Comment:
The set data is automatically generated so anything manually put here will get overwritten. I think it might be simpler to generate the proxy based on another property of the role until we get better images, like name, traits, or card number